v3/transport.go

```diff
@@ -80,8 +79,7 @@ func (t *Transport) RoundTrip(req *http.Request) (*http.Response, error) {
 		if err != nil {
 			log.Println(err)
 		} else {
-			os.Stderr.Write(dump)
-			os.Stderr.Write([]byte{'\n', '\n'})
+			log.Printf("%s", string(dump[:]))
```

Comment:
Any reasons for the type cast and the slice/copy?

Reply:
This turns out to be a type conversion not a type assertion (https://golang.org/ref/spec#Conversions)

The type conversion in this case requires a byte slice not a byte array, that's why I have to get the slice.

Reply:
Wouldn't `log.Printf` handle the conversion for us, since we're using asking for a string `%s`?

Reply:
Yeah...good call: https://play.golang.org/p/H1P5DQ6x9Kb